aiida/cmdline/commands/cmd_import.py

```diff
@@ -36,13 +37,108 @@ class ExtrasImportCode(Enum):
     ask = 'kca'
 
 
+def _try_import(first, **kwargs):
+    """
+    Utility function for `verdi import` to try-except to import archive
+
+    :param first: Boolean to determine what exception message to throw for IncompatibleArchiveVerisonError
```

Comment:
it is not obvious to me why this is called `first`.
Could it be given a more meaningful name (or a description that makes this name clear?).

Reply:
`first_try` or `first_run` or `migrated_import` or `already_migrated`? :)

Reply:
I'd vote for `migration_performed`

Reply:
Done